src/app/shared/components/forms/custom-form/custom-form.component.scss

```diff
@@ -8,3 +8,33 @@
     color: $color-white;
   }
 }
+
+label {
+  font-weight: 700;
+  font-size: 1.7rem;
+  line-height: 2.8rem;
+  color: $color-grey-dark;
+
+  .asterick {
```

Comment:
add this as common class and change name to something generic to be used for error msgs